libnmstate/ifaces/ifaces.py

```diff
@@ -440,6 +441,19 @@ def _validate_vrf_table_id_changes(self):
                         "is not supported yet"
                     )
 
+    def _validate_veth_peers(self):
+        for ifname, iface in self._kernel_ifaces.items():
+            if (
+                iface.type == InterfaceType.VETH
+                and iface.is_up
+                and not iface.peer
+            ):
+                if not self._cur_kernel_ifaces.get(iface.name):
+                    raise NmstateValueError(
+                        f"Veth interface {iface.name} must have a peer, "
+                        f"current value is {iface.peer}"
```

Comment:
the `iface.peer` is None.
How about `Veth interface {iface.name} does not exists, peer name is required for creating it`.

Reply:
Sure, let me do it.